rednotebook/journal.py

```diff
@@ -16,6 +16,20 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 # -----------------------------------------------------------------------
 
+from rednotebook.data import Month
+from rednotebook import storage
+from rednotebook.gui.main_window import MainWindow
+from rednotebook.util.statistics import Statistics
+from rednotebook import backup
+from rednotebook.util import dates
+from rednotebook import data
+from rednotebook import configuration
+from rednotebook import info
+from rednotebook.help import example_content
+from rednotebook.util import markup
+from rednotebook.util import utils
+from rednotebook.external import elibintl
+from rednotebook.util import filesystem
```

Comment:
These imports need to work even if the app is not installed. Please revert all changes to this file.